### PR TITLE
Add a simple GitHub Action to run tests

### PR DESCRIPTION
**Pull request by matyasaradi**

### CODE REVIEW

**Comment by matyasaradi** [comment]
Here is the list for supported Python versions:
https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

File: .github/workflows/renate-od.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: RENATE-OD CI
+
+on:
```
Comment (gergopokol) [inline .github/workflows/renate-od.yml:3]: Should run for every push, regardless of pull request, if possible.

Comment (gergopokol) [reply .github/workflows/renate-od.yml:3]: travis.yml should be removed.

Comment (gergopokol) [reply .github/workflows/renate-od.yml:3]: Both Pass and Fail should be tested.

Comment (matyasaradi) [reply .github/workflows/renate-od.yml:3]: > Should run for every push, regardless of pull request, if possible.

It is possible:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

PR is updated.

Comment (matyasaradi) [reply .github/workflows/renate-od.yml:3]: > travis.yml should be removed.

Removed.

**Review by gergopokol — APPROVED** [review]
Completed and tested

**Pull request merged**